pages/index.vue

```diff
@@ -86,13 +102,42 @@ const scales = {
   locrian
 };
 
```

Comment:
It would be nice to define intervals as constants holding the distance in half-tones.
```
MINOR_2ND = 1
MAJOR_2ND = 2
MINOR_3RD = 3
MAJOR_3RD = 4
PERFECT_4TH = 5
TRITONE = 6
PERFECT_5TH = 7
MINOR_6TH = 8
MAJOR_6TH = 9
MINOR_7TH = 10
MAJOR_7TH = 11
OCTAVE = 12
```